docs/shipping/Code/python.md

```diff
@@ -1343,9 +1343,288 @@ Replace `<<YOUR-APPLICATION-SCRIPT>>` with your Python application script name.
 
 Give your traces time to process, after which they'll be available in your [Tracing](https://app.logz.io/#/dashboard/jaeger) dashboard.
 
+## Python Application Setup for ECS Service with OpenTelemetry
```

Comment:
Perhaps worth to make tabs for the options like in node.js, because I see there are multiple options:
- Local host Python application auto instrumentation
- Docker Python application auto instrumentation
- Python Application Setup for ECS Service with OpenTelemetry
- Kuberenetes Python application auto insturmentation